src/AWS.Deploy.Common/IO/FileManager.cs

```diff
@@ -31,6 +34,12 @@ public class FileManager : IFileManager
         public Task WriteAllTextAsync(string filePath, string contents, CancellationToken cancellationToken) =>
             File.WriteAllTextAsync(filePath, contents, cancellationToken);
 
+        public FileStream OpenRead(string filePath) => File.OpenRead(filePath);
+
+        public string GetExtension(string filePath) => Path.GetExtension(filePath);
+
+        public long GetSizeInBytes(string filePath) => new FileInfo(filePath).Length;
```

Comment:
I'm worried about null pointer exceptions here. Can you do some sanity checks here?